packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx

```diff
@@ -0,0 +1,324 @@
+  const onClose = () => navigate('..');
+  const {
+    data: kubernetesCluster,
+    isPending: isPendingCluster,
+  } = useKubernetesCluster(projectId, kubeId);
+  const {
+    data: cloudSchema,
+    isPending: isPendingCloudSchema,
+  } = useGetCloudSchema();
+  const kubeVersions = cloudSchema.models['cloud.kube.VersionEnum'].enum;
+
+  const {
+    data: privateNetworks,
+    isPending: isPendingPrivateNetworks,
+  } = useAvailablePrivateNetworks(projectId, kubernetesCluster?.region);
+
+  const [formState, setFormState] = useState({
+    workerNodesPolicy: WORKER_NODE_POLICIES.DELETE,
+    selectedVersion: getFormatedKubeVersion(kubernetesCluster?.version),
+    privateNetworkId: kubernetesCluster?.privateNetworkId,
+    gateway: {
+      enabled: !!kubernetesCluster?.privateNetworkConfiguration
+        ?.defaultVrackGateway,
+      ip:
+        kubernetesCluster?.privateNetworkConfiguration?.defaultVrackGateway ||
+        '',
+    },
+  });
+  const { resetCluster, isPending: isPendingResetCluster } = useResetCluster({
+    onError(error: ApiError) {
+      addError(
+        <Translation ns="reset">
+          {(_t) =>
+            _t('pci_projects_project_kubernetes_service_reset_error', {
+              message: error?.response?.data?.message || error?.message || null,
+            })
+          }
+        </Translation>,
+        true,
+      );
+      onClose();
+    },
+    onSuccess() {
+      addSuccess(
+        <Translation ns="reset">
+          {(_t) => _t('pci_projects_project_kubernetes_service_reset_success')}
+        </Translation>,
+        true,
+      );
+      onClose();
+    },
+    projectId,
+    kubeId,
+    params: {
+      ...(formState.gateway.enabled && {
+        privateNetworkConfiguration: {
+          defaultVrackGateway: formState.gateway.ip,
+          privateNetworkRoutingAsDefault: true,
+        },
+      }),
+      privateNetworkId: formState.privateNetworkId,
+      version: formState.selectedVersion,
+      workerNodesPolicy: formState.workerNodesPolicy,
+    },
+  });
+
+  const isPending =
+    isPendingCluster ||
+    isPendingCloudSchema ||
+    isPendingPrivateNetworks ||
+    isPendingResetCluster;
+  return (
+    <OsdsModal
+      headline={tReset('pci_projects_project_kubernetes_service_reset')}
+      color={ODS_TEXT_COLOR_INTENT.warning}
+      onOdsModalClose={onClose}
+    >
+      <slot name="content">
+        {isPending ? (
+          <OsdsSpinner
+            inline
+            size={ODS_SPINNER_SIZE.md}
+            className="block text-center"
+            data-testid="renameCluster-spinner"
+          />
+        ) : (
+          <div className="mt-6">
+            <OsdsMessage type={ODS_MESSAGE_TYPE.warning}>
+              <span
+                dangerouslySetInnerHTML={{
+                  __html: tReset(
+                    'pci_projects_project_kubernetes_service_reset_message',
+                  ),
+                }}
+              />
+            </OsdsMessage>
```

Comment:
avoid `dangerouslySetInnerHTML` and split your translation message if you need differents styles. (e.g. normal and italic)

Reply:
We can't update translation message and we have to keep the existing translations